thoth/storages/graph/models.py

```diff
@@ -543,6 +543,8 @@ class AdviserRun(Base, BaseExtension):
     origin = Column(String(256), nullable=True)
     is_s2i = Column(Boolean, nullable=True)
     debug = Column(Boolean, nullable=False)
+    need_re_run = Column(Boolean, nullable=True)
+    re_run_adviser_id = Column(String(256), nullable=True)
```

Comment:
I think we can stick with Text, PostgreSQL treats these two with the same underlying logic so no downside when using Text.

Reply:
Right we have to move all to Text actually :) Thank!